code/game/objects/effects/spawners/structure.dm

```diff
@@ -31,3 +31,35 @@ again.
 						/obj/structure/grille,
 						/obj/structure/window/reinforced/fulltile
 						)
+
+/obj/effect/spawner/structure/window/reinforced/tinted
+	name = "tinted reinforced window spawner"
+	icon_state = "twindow_spawner"
+	spawn_list = 	list(
+						/obj/structure/grille,
```

Comment:
This formatting is yuck, could you change it for the file while you're here?
Drop the wall of indents; even the whole list on a single line would be fine, it's only two elements.